EIPS/eip-779.md

```diff
@@ -54,6 +54,8 @@ contract WithdrawDAO {
 }
 ```
 
+Blocks with block numbers in the range [1,920,000, 1,920,009] **MUST** have `0x64616f2d686172642d666f726b` (hex encoded ASCII string `dao-hard-fork`) in the `extraData` field of the block.
```

Comment:
```suggestion
Blocks with block numbers in the range [1,920,000 ... 1,920,009] **MUST** have `0x64616f2d686172642d666f726b` (hex encoded ASCII string `dao-hard-fork`) in the `extraData` field of the block.
```

Otherwise the punctuation conflicts with the separator.

Reply:
Or the alternative:
```suggestion
Blocks with block numbers in the range [1_920_000, 1_920_009] **MUST** have `0x64616f2d686172642d666f726b` (hex encoded ASCII string `dao-hard-fork`) in the `extraData` field of the block.
```

Reply:
I mildly favor the second since `[1234, 5678]` is standard math syntax for inclusive set range.